service/backend/cchainatomictx/backend.go

```diff
@@ -0,0 +1,71 @@
+package cchainatomictx
+
+import (
+	"github.com/ava-labs/avalanchego/codec"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/crypto"
+	"github.com/ava-labs/coreth/plugin/evm"
+	"github.com/coinbase/rosetta-sdk-go/types"
+
+	"github.com/ava-labs/avalanche-rosetta/client"
+	"github.com/ava-labs/avalanche-rosetta/mapper"
+	cmapper "github.com/ava-labs/avalanche-rosetta/mapper/cchainatomictx"
+	"github.com/ava-labs/avalanche-rosetta/service"
+)
+
+var (
+	_ service.ConstructionBackend = &Backend{}
+	_ service.AccountBackend      = &Backend{}
+)
+
+type Backend struct {
+	fac              *crypto.FactorySECP256K1R
+	cClient          client.Client
+	getUTXOsPageSize uint32
+	codec            codec.Manager
+	codecVersion     uint16
+	avaxAssetID      ids.ID
+}
+
+func NewBackend(cClient client.Client, avaxAssetID ids.ID) *Backend {
+	return &Backend{
+		fac:              &crypto.FactorySECP256K1R{},
+		cClient:          cClient,
+		getUTXOsPageSize: 1024,
+		codec:            evm.Codec,
+		codecVersion:     0,
```

Comment:
this should be filled with evm.codecVersion which is not exported. We should probably ask devs to export it, similarly to what it is done with pchain

Reply:
We can address it in a future PR after coreth exports the codec value.